python/main_loop.py

```diff
@@ -125,20 +143,28 @@ def createNewLocalPath(sailbot, maxAllowableRuntimeSeconds, desiredHeadingPublis
         isGlobalWaypointReached = (localPath.waypointReached(state.position, previousGlobalWaypoint, nextGlobalWaypoint,
                                                              True) or isLastWaypointReached)
 
+        if(lowWindConditions):
+            # ensure we havent updated local path recently before permitting
+            lowWindPermitsUpdate = (createNewLocalPathAttempts >= LOW_WIND_LOOPS_PER_PATH_UPDATE)
+            createNewLocalPathAttempts += 1
+        else:
+            lowWindPermitsUpdate = True
```

Comment:
We only call `createNewLocalPath()` when `mustUpdateLocalPath` or `generateNewLocalPathToCompare` is `True`, so would it make sense for the incrementing of `createNewLocalPathAttempts` to depend on these? Or should we stick with the number of loops approach?

Also, is there a reason you didn't reset `createNewLocalPathAttempts` when `lowWindconditions` is `False`?

Reply:
Good points, I think my next commit addresses both.